components/klab.component.random/src/main/java/org/integratedmodelling/random/adapters/RecreationIDBAdapter.java

```diff
@@ -175,14 +175,16 @@ private List<String> getStatesFromEnvelope(IEnvelope envelope) {
 
     private List<String> buildRecreationIDBInput(Map<String, String> parameters, IEnvelope envelope)
             throws UnsupportedEncodingException {
-        List<String> states = getStatesFromEnvelope(envelope);
+        // The list is initialized with the list of territories and DC, which cannot be queried by the same means as the states
+        List<String> regions = List.of("DC", "AS", "GU", "MP", "PR", "VI");
```

Comment:
Hi @diegomvd. I have initialized the list of regions to be queried from RecreationIDB, but I was wondering if some (or most) of these territories could be removed from the list. I assume D.C. is still relevant, but the rest are disconnected from the contiguous 48. Could you please check if they are needed?